dtm/remote.c

```diff
@@ -60,7 +60,7 @@ static struct m0_fop_type rem_rpc_fopt;
 static const struct m0_fop_type_ops rem_rpc_ftype_ops;
 static struct m0_rpc_item_type_ops rem_rpc_itype_ops;
 static const struct m0_rpc_item_ops rem_rpc_item_sender_ops;
-static const struct m0_rpc_item_ops rem_rpc_item_redo_ops;
+//static const struct m0_rpc_item_ops rem_rpc_item_redo_ops;
```

Comment:
Don't use C++ style comments.
Do  the same for all places.

Reply:
Replaced with M0_UNUSED macro, as it being not used anywhere else to suppress compiler warning